contracts/src/interfaces/INativeTokenDestination.sol

```diff
@@ -22,11 +22,6 @@ interface INativeTokenDestination is INativeTokenBridge {
      */
     event CollateralAdded(uint256 amount, uint256 remaining);
 
-    /**
-     * @dev Emitted when minting native tokens.
-     */
-    event NativeTokensMinted(address indexed recipient, uint256 amount);
```

Comment:
I removed this event all together because the native minter precompile already emits a `NativeCoinMinted` event, which felt duplicative and possibly confusing.

https://docs.avax.network/build/subnet/upgrade/customize-a-subnet#minting-native-coins